utils/yang_modules_setup.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+set -x
+# This is based on scripts/setup.sh from Netopeer2/libnetconf2
+# env variables NP2_MODULE_DIR, NP2_MODULE_PERMS must be defined and NP2_MODULE_OWNER, NP2_MODULE_GROUP will be used if
+# defined when executing this script!
+#if [ -z "$NP2_MODULE_DIR" -o -z "$NP2_MODULE_PERMS" ]; then
+#    echo "Required environment variables not defined!"
+#    exit 1
+#fi
+
+check_file_exists() {
+    file=$1
+
+    if [ ! -f "$file" ]; then
+        echo "Modules list file not found: $file"
+        exit 1
+    fi
+}
+
+MODULES_LIST_FILE_CONFD=${MODULES_LIST_FILE:-$(dirname "$0")/yang_modules_confd.inc}
```

Comment:
Didn't it work to have a variable called MODULES as before and use source on the file requested?